.gitlab-ci.yml

```diff
@@ -257,7 +257,8 @@ before_script:
   stage: stage-1
   interruptible: true
   variables:
-    PLATFORM: "https://github.com/coq/platform/archive/dev-ci.zip"
+    PLATFORM_SOURCES: "https://github.com/gares/platform/archive/override-pkg-url.zip"
+    PLATFORM_EXTRA_ARGS: '"-override-dev-pkg=coq=${CI_PROJECT_URL}#${CI_COMMIT_SHA}"'
```

Comment:
you need to use windows variable syntax, not $foo
(the artifacts name below is similarly bugged but nothing relies on it)